lua/terrortown/entities/items/item_ttt_radar/cl_init.lua

```diff
@@ -152,7 +152,7 @@ function RADAR:Draw(client)
 		surface.SetDrawColor(255, 255, 255, 200)
 
 		for _, bomb in pairs(self.bombs) do
-			if bomb.team == client:GetTeam() then
+			if bomb.team ~= nil and bomb.team == client:GetTeam() then
```

Comment:
I think the `~= nil` can be removed

Reply:
Could be removed, but this is more explicit